classes/Translations/Translations.php

```diff
@@ -56,6 +56,13 @@ public function getTranslations()
             'general' => [
                 'save' => $this->module->l('Save', 'translations'),
                 'testModeOn' => $this->module->l('Test mode is turned on', 'translations'),
+                'multiShop' => [
+                    'title' => $this->module->l('Multistore detected', 'translations'),
+                    'subtitle' => $this->module->l('You must configure your shops one by one for this service, but you will be able to use the same accounts.', 'translations'),
+                    'chooseOne' => $this->module->l('Please select the first shop to configure from the list below :', 'translations'),
+                    'group' => 'Group :',
+                    'tips' => $this->module->l('After you are done with the first shop, you can configure the others, by selecting them one by one in the shop selector, in the horizontal menu.', 'translations'),
```

Comment:
Another suggestion, to be reviewed by @LouiseBonnard & @Darmona 

```suggestion
                    'tips' => $this->module->l('When you are done with a shop, another one can be selected by using the shop selector at the top of the page.', 'translations'),
```

Reply:
```suggestion
                    'tips' => $this->module->l('Once you are done with the first shop, you can configure the others: select them one by one with the shop selector, in the horizontal menu.', 'translations'),
```

Reply:
Done, thanks for help